test/framework/src/main/java/org/elasticsearch/test/AbstractQueryTestCase.java

```diff
@@ -308,7 +308,7 @@ public void testUnknownField() throws IOException {
         } while (testQuery.toString().contains(marker));
         testQuery.queryName(marker); // to find root query to add additional bogus field there
         String queryAsString = testQuery.toString().replace("\"" + marker + "\"", "\"" + marker + "\", \"bogusField\" : \"someValue\"");
-        ParsingException e = expectThrows(ParsingException.class, () -> parseQuery(queryAsString));
+        RuntimeException e = expectThrows(RuntimeException.class, () -> parseQuery(queryAsString));
```

Comment:
Maybe leave a comment that this is here for the duration of the migration or something. Could you add an assertion that the HTTP status code is still 400?